lib/maple_tech/technologies/version.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module MapleTech
+  class Technologies
+    VERSION = '0.1'
+  end
```

Comment:
why is this needed?

Reply:
to maintain versions, by default gems add a versions.